modules/host/consts.go

```diff
@@ -78,10 +78,10 @@ var (
 	defaultCollateralBudget = types.SiacoinPrecision.Mul64(100e3)
 
 	// defaultContractPrice defines the default price of creating a contract
-	// with the host. The default is set to 30 siacoins, which the file
-	// contract revision can have 15 siacoins put towards it, and the storage
-	// proof can have 15 siacoins put towards it.
-	defaultContractPrice = types.SiacoinPrecision.Mul64(3) // 3 siacoins
+	// with the host. The current default is 0.05. This was chosen since it is
+	// 1/2e6 times 10,000 bytes. 1/2e6 is the current transaction pool's fee
+	// estimation minimum.
+	defaultContractPrice = types.SiacoinPrecision.Div64(200) // 0.05 siacoins
```

Comment:
Going to want to adjust this since the min fee for the tpool is not coming down.

Reply:
Though, it's probably fine to have the default contract price minimum be something like half of the lowest value that the tpool will ever spit out.